core_java/jdbc/src/main/java/ca/jrvs/apps/jdbc/JDBCExecutor.java

```diff
@@ -0,0 +1,8 @@
+package ca.jrvs.apps.jdbc;
```

Comment:
Suggestion: you should implement CRUD in this class. If you have implement Junit Test, skip this suggestion please.